test/rdf-components-mocha.js

```diff
@@ -49,6 +52,21 @@ describe('rdf components', function() {
             });
         }).should.eventually.include.keys('triples');
     });
+    it("should round trip a graph", function() {
+        return test.createNetwork({
```

Comment:
Can we add some comments being really clear what's being done?  If I understand right, I think we are creating a 2 node graph here - one with the rdfLoad component and one with rdfJsonld - correct?    A comment like 
    // creating 2 node graph (rdfLoad & rdfJsonld components)